torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -225,6 +222,29 @@ def clamp_bounding_box(
     return convert_format_bounding_box(xyxy_boxes, old_format=BoundingBoxFormat.XYXY, new_format=format, inplace=True)
 
 
+def clamp_bounding_box(
+    inpt: datapoints.InputTypeJIT,
+    format: Optional[BoundingBoxFormat] = None,
+    spatial_size: Optional[Tuple[int, int]] = None,
+) -> datapoints.InputTypeJIT:
+    if not torch.jit.is_scripting():
+        _log_api_usage_once(clamp_bounding_box)
+
+    if torch.jit.is_scripting() or is_simple_tensor(inpt):
+        if format is None or spatial_size is None:
+            raise RuntimeError("For simple tensor inputs, `format` and `spatial_size` has to be passed.")
```

Comment:
Don't we raise `ValueError` for anything related to user-input?
Also, should we add a small test somewhere that makes sure this exception and the one below are properly hit?